parquet/src/main/java/org/apache/iceberg/parquet/MessageTypeToType.java

```diff
@@ -229,7 +229,7 @@ private void addAlias(String name, int fieldId) {
     aliasToId.put(DOT.join(path(name)), fieldId);
   }
 
-  protected int nextId() {
+  private int nextId() {
```

Comment:
I'd also prefer to revert this change. There may not be a use of it as a `protected` method any more, but we don't need to change this file and risk git conflicts.

Reply:
OK